package.json

```diff
@@ -16,10 +16,10 @@
   "scripts": {
     "codegen": "turbo codegen",
     "create-block": "yarn workspace @local/repo-chores exe scripts/create-block.ts",
-    "dev": "NODE_ENV=development turbo dev --log-order stream --filter '@apps/hash-api' --filter '@apps/hash-search-loader' --filter '@apps/hash-realtime' --filter '@apps/hash-frontend' --",
+    "dev": "turbo dev --log-order stream --filter '@apps/hash-api' --filter '@apps/hash-search-loader' --filter '@apps/hash-realtime' --filter '@apps/hash-frontend' --",
     "dev:backend": "NODE_ENV=development turbo dev --log-order stream --filter '@apps/hash-api' --filter '@apps/hash-realtime' --filter '@apps/hash-search-loader' --",
     "dev:backend:api": "NODE_ENV=development turbo dev --log-order stream --filter '@apps/hash-api' --",
-    "dev:backend:realtime": "NODE_ENV=development turbo dev --log-order stream --filter '@apps/hash-realtime' --",
+    "dev:backend:realtime": "turbo dev --log-order stream --filter '@apps/hash-realtime' --",
```

Comment:
Not superfluous, no, but these `NODE_ENV` were added in https://github.com/hashintel/hash/pull/3096 and have caused `realtime` to start logging out a bunch of stuff that was annoying, so I imagine it wasn't being run with `NODE_ENV=development` before. I can revert this and we can figure out a better way of managing it. I think probably the `NODE_ENV=development` should be in the individual app's package.jsons rather than here in any case

Reply:
Ah makes sense, I'm definitely in favour of getting rid of those realtime logs so happy to remove this here for now.

Reply:
I did it less hackily in https://github.com/hashintel/hash/pull/3126/commits/f0632f955d1b9c15f334fa12c77a0009ded11f71